freetextresponse/freetextresponse.py

```diff
@@ -405,6 +407,7 @@ def _compute_score(self):
         )
 
     def _determine_credit(self):
+        # pylint: disable=redefined-variable-type
```

Comment:
This is an issue with how pylint handles enums/conditionals.   I think it is fixed in a newer version.

Redefinition of result type from freetextresponse.freetextresponse.Credit.zero to freetextresponse.freetextresponse.Credit.full